applications/asset_tracker/src/main.c

```diff
@@ -759,6 +769,7 @@ static void button_send(bool pressed)
 }
 #endif
 
+#if defined(CONFIG_MOTION)
```

Comment:
It looks like this line can be combined with the line below in one ```#if defined```.

Reply:
It cant as far as I can see. The #if defined (CONFIG_MOTION) spans more functions than CONFIG_GPS_START_ON_MOTION does?

Reply:
Yes. Indeed it does...